### PR TITLE
Adds tests for GET /members API

### LINKED ISSUE #164: Add tests for GET /members API

### PR DESCRIPTION
**Pull request by swarajpure**
Fixes #164

### CODE REVIEW

File: test/integration/membersController.test.js [hunk 1]
```diff
@@ -0,0 +1,33 @@
...
+
+chai.use(chaiHttp)
+
+describe('Members', function () {
+  before(async function () {
+    await addUser()
+  })
+
+  describe('GET /members', function () {
+    it('Get all the members in the database', function (done) {
```
Comment (ankurnarkhede) [inline test/integration/membersController.test.js:16]: Can you also add a test when there are no members in DB?

**Review by ankurnarkhede — CHANGES_REQUESTED** [review]

File: test/integration/membersController.test.js (hunk 1 above)
Comment (swarajpure) [reply test/integration/membersController.test.js:16]: We had not handled the case when there are no members in the DB. Made the change here: https://github.com/Real-Dev-Squad/website-backend/pull/167
Please review

File: test/integration/tasksController.test.js [hunk 2]
```diff
@@ -146,7 +146,7 @@ describe('Tasks', function () {
           expect(res).to.have.status(200)
           expect(res.body).to.be.a('array')
           expect(res.body).to.have.length.above(0)
-          res.body.map((task) => {
+          res.body.forEach((task) => {
```
Comment (swarajpure) [inline test/integration/tasksController.test.js:149]: Reason: 
image

**Review by ankurnarkhede — APPROVED** [review]
LGTM ✅

**Pull request merged**